src/pages/AllDao.vue

```diff
@@ -362,7 +362,7 @@ export default defineComponent({
 							key: index,
 							dao_id: +item.appId,
 							token_id: +item.assetId,
-							token_name: tokenData.name as string,
+							token_name: tokenData.an as string,
```

Comment:
Why is it `an`

Reply:
We are now directly using the PostreSQL for fetching data. Indexer Client used to do these proper naming before it returns the response on client side but now we are fetching directly. This is how it is stored in PostreSQL.